dae/dae/variants_loaders/cnv/tests/test_cnv_loader.py

```diff
@@ -111,26 +225,71 @@ def test_cnv_loader_alt_best_state(fixture_dirname, gpf_instance_2013):
         for fv in _fvs:
             fvs.append(fv)
 
-    assert len(svs) == 1
+    assert len(svs) == 4
     assert len(fvs) == 4
     print(fvs[0].best_state)
 
 
-def test_cnv_loader_alt_2(fixture_dirname, gpf_instance_2013):
-    families_file = fixture_dirname("backends/cnv_ped.txt")
-    families = FamiliesLoader.load_simple_families_file(families_file)
+@pytest.fixture(scope="module")
+def variants_file_alt_2(
+    tmp_path_factory: pytest.TempPathFactory
+) -> Path:
+    root_path = tmp_path_factory.mktemp("acgt_instance")
+    cnv_path = setup_denovo(
+        root_path / "cnv_data" / "in.tsv",
+        """
+        person_id  location     variant
+        f1.mo      chr1:1-3	    duplication
+        f2.fa      chr1:5-8	    duplication
+        f1.fa      chr2:11-13   deletion
+        f2.p1      chr2:9-10    deletion
+        f1.s1      chr1:15-17   deletion
+        f1.fa      chr1:20-21   deletion
+        f1.s1      chr2:23-26   deletion
+        f2.mo      chr1:29-31	deletion
+        f2.mo      chr2:33-35	duplication
+        f1.fa      chr2:38-40	duplication
+        f2.mo      chr2:41-43	deletion
+        f1.fa      chr1:44-46	deletion
+        f1.s1      chr1:48-50	duplication
+        f2.mo      chr2:53-55	duplication
+        f1.s1      chr2:57-60	duplication
+        f1.p1      chr1:18-19	deletion
+        f2.fa      chr2:61-64	deletion
+        f2.mo      chr2:68-71	deletion
+        f1.fa      chr1:73-75	deletion
+        f1.mo      chr2:77-79	duplication
+        f1.mo      chr2:81-83	deletion
+        f1.fa      chr2:86-90	deletion
+        f1.mo      chr1:91-93	deletion
+        f2.fa      chr1:94-95	duplication
+        f1.fa      chr2:96-97	deletion
+        f1.p1      chr2:98-99	deletion
+        f1.s1      chr1:51-52	duplication
+        f2.mo      chr1:65-67	deletion
+        f2.fa      chr2:84-85	duplication
+        f2.mo      chr1:39-40	deletion
+        """)
```

Comment:
These are too many variants. We do not need so many variants for the test.

Change this fixture to include ~4-5 CNV variants.